spark/src/main/scala/org/apache/comet/serde/arrays.scala

```diff
@@ -126,6 +126,21 @@ object CometArraysOverlap extends CometExpressionSerde with IncompatExpr {
   }
 }
 
+object CometArrayExcept extends CometExpressionSerde with CometExprShim {
```

Comment:
Any reason for using `CometExprShim`?
Also not sure how ready this is regarding Spark compatibility although it seems the test coverage is good. Adding `IncompatExpr` might be an option.

Reply:
Added and fixed the tests.